File: problemas/iniciante/2862/2862.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = '';
let lineCount = 0;

process.stdin.on('data', (chunk) => {
  input += chunk;
});

process.stdin.on('end', () => {
  var lines = input.split('\n');

  let C = parseInt(lines.shift()!);
  for (let i = 0; i < C; ++i) {
    let N = parseInt(lines.shift()!);

    if (N > 8000) {
      console.log('Mais de 8000!');
    } else {
      console.log('Inseto!');
    }
  }
});

File: problemas/iniciante/2862/2862.dart
import 'dart:io';

void main() {
  int c = int.parse(stdin.readLineSync()!);
  for (int i = 0; i < c; i++) {
    int n = int.parse(stdin.readLineSync()!);

    if (n <= 8000) {
      print("Inseto!");
    } else {
      print("Mais de 8000!");
    }
  }
}
